api_server/src/main/java/io/zetch/app/web/HelloController.java

```diff
@@ -1,13 +1,27 @@
 package io.zetch.app.web;
 
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class HelloController {
```

Comment:
Is this controller for just testing??

Reply:
Yes, I kept it in as an example of public/private routes. We can clean up later